Data-Structure-ARRAYS.md

```diff
@@ -0,0 +1,62 @@
+#Arrays
+
+`array` is a kind of data structure that can store a fixed-size sequential collection of elements of the same type. An `array` is used to store 
+a collection of data, but it is often more useful to think of an `array` as a collection of variables of the same type.
+
+`array` consists of contiguous memory locations. The lowest address corresponds to the first element and the highest address to the last 
```

Comment:
Preface with _Internally_, or _under the hood_. But remember, languages like Python might not have this as true. Heck, any language that runs on a VM (Java, Ruby, Python, JS etc.), you probably cannot guarantee that.

But these languages still implement Arrays.